.gitignore

```diff
@@ -0,0 +1,6 @@
+.gitignore
+.venv
+.idea
+db.sqlite3
+__pycache__
```

Comment:
we have to separate terms of local and global gitignore. Local blocks to push cache, temp files, databases and so on. Global blocks to pull github specific, workflows and so on.

Reply:
Add only files that are create during work of python and Django framework and pycharm project.